src/services/proxy.rs

```diff
@@ -38,14 +38,16 @@ impl Service {
     /// * The image URL is not an image.
     /// * The image is too big.
     /// * The user quota is met.
-    #[allow(clippy::missing_panics_doc)]
     pub async fn get_image_by_url(&self, url: &str, maybe_user_id: Option<UserId>) -> Result<Bytes, Error> {
+        let Some(user_id) = maybe_user_id else {
+            return Err(Error::Unauthenticated);
```

Comment:
Hi @mario-nt, should not this also be `ServiceError::UnauthorizedActionForGuests`?

Reply:
Hi @josecelano, in this case, I didn't want to change the function's signature and also, that error it is already implemented and returns the text "Sign in to see image":

https://github.com/torrust/torrust-index/blob/720b2fa537676fc2b34c3c0b986599c12e695a61/src/web/api/server/v1/contexts/proxy/handlers.rs#L30-L39

https://github.com/torrust/torrust-index/blob/720b2fa537676fc2b34c3c0b986599c12e695a61/src/ui/proxy.rs#L19-L46